charts/fleet/templates/ingress.yaml

```diff
@@ -4,9 +4,9 @@ kind: Ingress
 metadata:
 {{- if or .Values.ingressAnnotations .Values.gke.useGKEIngress }}
   annotations:
-  {{- with .Values.ingressAnnotations }}
-  {{ toYaml . | trim | indent 2 }}
-  {{- end }}
+{{- range $key, $value := $.Values.ingressAnnotations }}
+    {{ $key }}: {{ $value | quote }}
+{{- end }}
```

Comment:
This looks like it should do effectively the same thing... Can you explain the motivation for this change?

Reply:
the previos block doesn't set correct spaces when you modify the annotations 
example
values.yml
```
    ingressAnnotations:
      kubernetes.io/ingress.class: nginx
      cert-manager.io/cluster-issuer: letsencrypt
```

The ingress file will be 

```
    annotations:
      kubernetes.io/ingress.class: nginx
    cert-manager.io/cluster-issuer: letsencrypt
```

Reply:
Makes sense. Thank you.